pyscriptjs/src/pyscript.py

```diff
@@ -388,29 +390,55 @@ def revert(self):
     def write(self, txt):
         console.log("writing to", self._out, txt, self._append)
         if self._out:
-            pyscript.write(self._out, txt, append=self._append)
+            out_element = document.querySelector(f"#{self._out}")
+            out_element_id = self._out
+
+            if not out_element:
+                return
+
+            if self._append:
+                child = document.createElement("div")
+                exec_id = out_element.childElementCount + 1
+                out_element_id = child.id = f"{self._out}-{exec_id}"
+                out_element.appendChild(child)
+
+            out_element = document.querySelector(f"#{out_element_id}")
+
+            html, mime_type = format_mime(txt)
+            if mime_type in ("application/javascript", "text/html"):
+                script_element = document.createRange().createContextualFragment(html)
+                out_element.appendChild(script_element)
+            else:
+                if html == "\n":
+                    return
+                out_element.innerHTML = html
+
```

Comment:
Why the logic here and not in the write method? 

Ideally, I'd like this logic to be reused un upstream objects/functions so that users can use it in their own code as well.

Reply:
Good point, I also thought about this. Currently, the REPL uses the `OutputCtxManager.write` method to output, not the `Element.write` method.
If you want I can replace the `Element.write` method with this functionality and let `OutputCtxManager` use it. That should be of higher reusablity down the line.

Reply:
Added a commit for this.
I tested it with a few examples. But I think this should be tested heavily before merging, as it affects every output operation.